src/compiler/symb-keyword.bas

```diff
@@ -482,7 +482,12 @@ sub symbKeywordTypeInit( )
 		'' end type
 		symbStructEnd( s )
 
-		'' type cva_list as __va_list_tag alias "__builtin_va_list"
+		'' subtype mangle modifier
+		s = symbCloneSymbol( s )
```

Comment:
What's the purpose of the cloning here? I couldn't find anything still using the original symbol, so it seemed unused. In `cMangleModifier()` it makes more sense to me; it probably has to avoid marking the original type with the `FB_UDTOPT_ISVALISTSTRUCT*` flags because otherwise it would be emitted incorrectly.